.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,6 +26,65 @@
 <!-- How is this issue solved/fixed? What is the design? -->
 <!-- It's better if we elaborate -->
 
+### Server checklist
+<!-- A checklist for server code -->
+
+#### Catalog upgrade
+<!-- Is hdb_catalog version bumped? -->
+Is Hasura catalogue version bumped?
+- [ ] Yes
+  - [ ] Update docs with down SQL script
+- [ ] No
+
+#### Metadata
+<!-- Hasura metadata changes -->
+
+Any new metadata feature is added?
+
+- [ ] Yes;
+   Does `run_sql` auto manages the new metadata through schema diffing?
+  - [ ] Yes
+  - [ ] Not required
+- [ ] No
+
+#### Breaking changes
+
+- [ ] No Breaking changes
+
+1. Metadata API
+
+   Existing `query` types:-
+   - [ ] Modify `args` payload which is not backward compatible
+   - [ ] Behavioural change of the API
+   - [ ] Change in response `JSON` schema
+   - [ ] Change in error code
+   <!-- Add if anything not listed above -->
+
+2. GraphQL API
+
+   Schema Generation:-
+   <!-- Any changes in schema auto-generation logic -->
+   <!-- All GraphQL schema names are case sensitive -->
+   - [ ] Change in any `NamedType`
+   - [ ] Change in table field names
+     <!-- Add if anything not listed above -->
+
+   Schema Resolve:-
+   <!-- Any change in logic of resolving input request -->
+   - [ ] Change in treatment of `null` value for input fields
```

Comment:
This seems like an oddly specific bullet point to me, but maybe I’m just not understanding it? Is there a reason this in particular is worth calling out above other changes?

Reply:
[GraphQL spec](https://graphql.github.io/graphql-spec/June2018/#sec-Null-Value) says explicitly providing `null` value and omitting the input field are similar but not identical.
For example, in insert mutation, the objects are inserted with `DEFAULT` value for omitted fields and `NULL` value for `null` fields. Change in treatment of `null` value may break the behaviour. This can be a "soft" breaking change. 

I'll rephrase the sentence to `.. for any input fields`